### ISSUE TITLE
Cannot use "Bio Formats Importer" FIJI plugin

### ISSUE THREAD

**Issue by tomerfried** [imagej/pyimagej#133]
I'm trying to use "Bio Formats Importer" by running macros with pyimagej. this is the code:

`import imagej
from scyjava import jimport
from skimage import io

ij = imagej.init('C:/Users/Tomer/Desktop/Hereon Project/Fiji.app')
WindowManager = jimport('ij.WindowManager')
macrotest = """
run("Bio-Formats Importer", "open=[C:/Users/Tomer/Desktop/Hereon Project/pyimagej/a.tif] autoscale color_mode=Default rois_import=[ROI manager] view=Hyperstack stack_order=XYCZT");
"""
ij.py.run_macro(macrotest)
img = WindowManager.getCurrentImage()
ij.py.show(img)`

and this is the error:

[java.lang.Enum.toString] Macro Error: Unrecognized command: "Bio-Formats Importer" in line 2
 
run ( "Bio-Formats Importer" , "open=[C:/Users/Tomer/Desktop/Hereon Project/pyimagej/a.tif] autoscale color_mode=Defaul...[java.lang.Enum.toString] 

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
~\AppData\Local\Temp/ipykernel_3740/809048045.py in <module>
      6 ij.py.run_macro(macrotest)
      7 img = WindowManager.getCurrentImage()
----> 8 ij.py.show(img)

~\Desktop\Hereon Project\pyimagej\lib\site-packages\imagej\__init__.py in show(self, image, cmap)
    608             """
    609             if image is None:
--> 610                 raise TypeError('Image must not be None')
    611 
    612             # NB: Import this only here on demand, rather than above.

Iv'e tried using run_plugin() but the error is the same.

Is there a solution for this? because I didn't find any in previous topics online.
Basically, all I need is a good images importer for tiff files, captured from a microscope for a biological research.
It is important that there would not be a loss of data when importing.

I'd like to hear any alternatives for Bio Formats if you guys know something

Thank you

**Comment by ctrueden** [comment]
@tomerfried This is probably the same issue as #129. I'll investigate when I have time!

**Comment by hinerm** [comment]
@ctrueden @elevans so far I have confirmed that there is a negative interaction with the Bio-Formats `plugins.config`. The importer sets [`Location=[Local machine] windowless=false`](https://github.com/ome/bioformats/blob/b67dfcac19b898e9a8e2481e5288bdc4b7aa782d/components/bio-formats-plugins/src/plugins.config#L34) and if I instead pass an empty string, the Importer command is suddenly discovered within PyImageJ.

Other commands are similarly missing: [Bio-Formats](https://github.com/ome/bioformats/blob/b67dfcac19b898e9a8e2481e5288bdc4b7aa782d/components/bio-formats-plugins/src/plugins.config#L28), [Bio-Formats (Windowless)](https://github.com/ome/bioformats/blob/b67dfcac19b898e9a8e2481e5288bdc4b7aa782d/components/bio-formats-plugins/src/plugins.config#L30) and [Bio-Formats Windowless Importer](https://github.com/ome/bioformats/blob/b67dfcac19b898e9a8e2481e5288bdc4b7aa782d/components/bio-formats-plugins/src/plugins.config#L39) (the latter two being the same command)

Oddly, the [Bio-Formats (Remote)](https://github.com/ome/bioformats/blob/b67dfcac19b898e9a8e2481e5288bdc4b7aa782d/components/bio-formats-plugins/src/plugins.config#L29) commands are both present.

To summarize the state of `plugins.config`: 
*  Commands with `Location=[Local machine]` are not found, but `Location=[Internet]` are (problem a space in the value?)
* If there is a trailing space in the string the command is not discovered (`"Location=[Internet]"` is OK but not `"Location=[Internet] "`) 💥 

So I guess the next step is to understand where these params are actually being parsed and interpreted. 😿  Maybe it can be fixed with a `trim()` and
